patches/minecraft/net/minecraft/client/renderer/ItemInHandRenderer.java.patch

```diff
@@ -11,7 +11,7 @@
          if (iteminhandrenderer$handrenderselection.renderOffHand) {
              float f6 = interactionhand == InteractionHand.OFF_HAND ? f : 0.0F;
              float f7 = 1.0F - Mth.lerp(p_109315_, this.oOffHandHeight, this.offHandHeight);
-+            if (!net.minecraftforge.client.ForgeHooksClient.renderSpecificFirstPersonHand(InteractionHand.MAIN_HAND, p_109316_, p_109317_, p_109319_, p_109315_, f1, f6, f7, this.mainHandItem))
++            if (!net.minecraftforge.client.ForgeHooksClient.renderSpecificFirstPersonHand(InteractionHand.OFF_HAND, p_109316_, p_109317_, p_109319_, p_109315_, f1, f6, f7, this.mainHandItem))
```

Comment:
Ah you're right, a copy paste error.
This also needs to change `this.mainHandItem` to `this.offHandItem`